bubble_sort_test.rb

```diff
@@ -4,32 +4,12 @@
 require_relative './bubble_sort'
 
 class BubbleSortTest < Minitest::Test
-  def test_bubblesort_class
-    sorter = SortingSuite::BubbleSort.new
-    assert_equal "SortingSuite::BubbleSort", sorter.class.to_s
-  end
-
-  def test_bubblesort_sort_accepts_1_arg
-    sorter = SortingSuite::BubbleSort.new
-    assert_equal 1, sorter.method(:sort).arity
-  end
-
   def test_bubblesort_sort_returns_array
     sorter = SortingSuite::BubbleSort.new
     numbers = []
-    assert_equal "Array", sorter.sort(numbers).class.to_s
-  end
-
-  def test_bubblesort_should_swap
-    sorter = SortingSuite::BubbleSort.new
-    numbers = [2, 1]
-    assert_equal true, sorter.should_swap?(numbers, 0)
-  end
-
-  def test_bubblesort_swap_two_elements
```

Comment:
There's a rule in TDD that you should only test the public interfaces. What this means is that methods that you write that are internal to the class shouldn't be tested. Since you're only using the `swap` and `should_swap?` methods in the `sort` method, you're already testing them with the tests below that test the `sort` method, and these tests aren't strictly needed.